### PR TITLE
add appinsights logging

### PR DESCRIPTION
**Pull request by ryandac**
There is no logging in aat for this service. Could this be why?

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -176,6 +176,8 @@ dependencies {
 
   compile group: 'uk.gov.hmcts.reform', name: 'health-spring-boot-starter', version: '0.0.4'
   compile group: 'uk.gov.hmcts.reform', name: 'java-logging', version: versions.reformLogging
+  compile group: 'uk.gov.hmcts.reform', name: 'java-logging-httpcomponents', version: versions.reformLogging
```
Comment (doncem) [inline build.gradle]: not a requirement ;)

**Comment by ryandac** [comment]
@doncem What is agent? 2.3.1 of agent, is it? What should I put in web.config? I have never seen that file before.

**Comment by doncem** [comment]
inspect this PR hmcts/bulk-scan-processor#613 for references to `java-logging-appinsights` and you can find agent jar which you require [here](https://github.com/Microsoft/ApplicationInsights-Java/releases/tag/2.3.1). haven't checked all, but please do go through app config is missing. linked PR should be sufficient info

p.s. probably want [that](https://github.com/hmcts/sscs-ccd-callback-orchestrator/blob/master/src/main/resources/application.yaml#L11) properly capitalised/named 😉

**Comment by ryandac** [comment]
@doncem I've changed the PR based on your comments. Does it seem okay now?

File: lib/AI-Agent.xml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      <Method name="send"/>
+    </Class>
+    <Class name="com.microsoft.azure.storage.blob.CloudBlobClient">
+      <Method name="listContainers" signature="()Ljava/lang/Iterable;"/>
+    </Class>
+    <Class name="com.microsoft.azure.storage.blob.CloudBlob">
+      <Method name="acquireLease"
+              signature="(Ljava/lang/Integer;Ljava/lang/String;)Ljava/lang/String;"/>
+      <Method name="deleteIfExists" signature="()Z"/>
+    </Class>
```
Comment (doncem) [inline lib/AI-Agent.xml]: perhaps you don't need those at all 😂:wink:

**Pull request merged**